src/meta/base_values.yaml

```diff
@@ -0,0 +1,514 @@
+minecraft:end_stone: 20
+minecraft:dragon_egg: -1
+
+# Interactive blocks
+minecraft:chipped_anvil: 800
+minecraft:damaged_anvil: 400
+minecraft:bell: 200
+minecraft:bee_nest: 200
+
+# Terracotta
+minecraft:terracotta: -1
+
+# Shulker boxes (disabled for safety)
+minecraft:white_shulker_box: -1
+minecraft:orange_shulker_box: -1
+minecraft:magenta_shulker_box: -1
+minecraft:light_blue_shulker_box: -1
+minecraft:yellow_shulker_box: -1
+minecraft:lime_shulker_box: -1
+minecraft:pink_shulker_box: -1
+minecraft:gray_shulker_box: -1
+minecraft:light_gray_shulker_box: -1
+minecraft:cyan_shulker_box: -1
+minecraft:purple_shulker_box: -1
+minecraft:blue_shulker_box: -1
+minecraft:brown_shulker_box: -1
+minecraft:green_shulker_box: -1
+minecraft:red_shulker_box: -1
+minecraft:black_shulker_box: -1
+
+# Terracotta
+minecraft:white_glazed_terracotta: 15
+minecraft:orange_glazed_terracotta: 15
+minecraft:magenta_glazed_terracotta: 15
+minecraft:light_blue_glazed_terracotta: 15
+minecraft:yellow_glazed_terracotta: 15
+minecraft:lime_glazed_terracotta: 15
+minecraft:pink_glazed_terracotta: 15
+minecraft:gray_glazed_terracotta: 15
+minecraft:light_gray_glazed_terracotta: 15
+minecraft:cyan_glazed_terracotta: 15
+minecraft:purple_glazed_terracotta: 15
+minecraft:blue_glazed_terracotta: 15
+minecraft:brown_glazed_terracotta: 15
+minecraft:green_glazed_terracotta: 15
+minecraft:red_glazed_terracotta: 15
+minecraft:black_glazed_terracotta: 15
+
+# Concrete
+minecraft:white_concrete: 12
+minecraft:orange_concrete: 12
+minecraft:magenta_concrete: 12
+minecraft:light_blue_concrete: 12
+minecraft:yellow_concrete: 12
+minecraft:lime_concrete: 12
+minecraft:pink_concrete: 12
+minecraft:gray_concrete: 12
+minecraft:light_gray_concrete: 12
+minecraft:cyan_concrete: 12
+minecraft:purple_concrete: 12
+minecraft:blue_concrete: 12
+minecraft:brown_concrete: 12
+minecraft:green_concrete: 12
+minecraft:red_concrete: 12
+minecraft:black_concrete: 12
+
+# Coral
+minecraft:dead_tube_coral_block: 1
+minecraft:dead_brain_coral_block: 1
+minecraft:dead_bubble_coral_block: 1
+minecraft:dead_fire_coral_block: 1
+minecraft:dead_horn_coral_block: 1
+minecraft:tube_coral_block: 20
+minecraft:brain_coral_block: 20
+minecraft:bubble_coral_block: 20
+minecraft:fire_coral_block: 20
+minecraft:horn_coral_block: 20
+minecraft:tube_coral: 20
+minecraft:brain_coral: 20
+minecraft:bubble_coral: 20
+minecraft:fire_coral: 20
+minecraft:horn_coral: 20
+minecraft:dead_brain_coral: 1
+minecraft:dead_bubble_coral: 1
+minecraft:dead_fire_coral: 1
+minecraft:dead_horn_coral: 1
+minecraft:dead_tube_coral: 1
+minecraft:tube_coral_fan: 20
+minecraft:brain_coral_fan: 20
+minecraft:bubble_coral_fan: 20
+minecraft:fire_coral_fan: 20
+minecraft:horn_coral_fan: 20
+minecraft:dead_tube_coral_fan: 1
+minecraft:dead_brain_coral_fan: 1
+minecraft:dead_bubble_coral_fan: 1
+minecraft:dead_fire_coral_fan: 1
+minecraft:dead_horn_coral_fan: 1
+
+# Transportation
+minecraft:saddle: 200
+minecraft:elytra: 5000
```

Comment:
Some insanely rare items, like elytra, could give more. This is basically worth 2 emerald ores, or 2 ancient debris, which seems low.

This applies to some other more rare items as well